components/TestimonyCard/TestimonyItem.tsx

```diff
@@ -250,6 +238,26 @@ export const TestimonyItem = ({
               </FooterButton>
             </Col>
           )}
+
+          {isUser && (
+            <Col>
+              <FooterButton variant="link">
+                <Internal
+                  className={styles.link}
+                  href={formUrl(testimony.billId, testimony.court)}
+                >
+                  Edit
```

Comment:
![image](https://github.com/codeforboston/maple/assets/73559781/b224dd9f-cf78-461b-b6de-fccb09e85145)

the new Edit:

1) is positioned a little higher than the other options -> be good to get those consistent

2) there seem to be 2 Edit options per item.  This seems to occur on desktop and not mobile.